extensions/notion/src/components/PageListItem.tsx

```diff
@@ -121,9 +121,21 @@ export function PageListItem({
     <Action title="Open in Notion" icon="notion-logo.png" onAction={() => handleOnOpenPage(page, setRecentPage)} />
   );
 
+  const { open_in } = getPreferenceValues<Preferences>();
+  const otherNotionActionTitle = open_in === "web" ? "Open in App" : "Open in Browser";
+  const otherNotionAction = open_in === "web" ? "app" : "web";
+  const openInNotionOtherAction = (
```

Comment:
What about adding the `alternate` name to the variable as you did in the PR's title?